### PR TITLE
Add utils for filling input and selects

### PR DESCRIPTION
**Pull request by marthacryan**
Adds `fillInput` and `updateSelectedValue` functions, which takes the `title` of an input (which is assumed to be in a `.spacing-row` div with the input), and fills with the given `value`. This allows for more precision and cleaner code when looking for inputs to fill. This allows us to avoid relying on things like the "nth" input.

### CODE REVIEW

File: tests/streamlit_ui_tests/utils.ts [hunk 1]
```diff
@@ -48,6 +48,15 @@ export const checkColumnCellsHaveExpectedValues = async (mito: FrameLocator, col
...
 
+export const fillInput = async (mito: FrameLocator, title: string, value: string): Promise<void> => {
+    await mito.locator('.spacing-row', { hasText: title }).locator('input').fill(value);
+}
+
+export const updateSelectedValue = async (mito: FrameLocator, title: string, value: string): Promise<void> => {
+    await mito.locator('.spacing-row', { hasText: title }).locator('.select-text').click();
+    await mito.locator('.mito-dropdown-item', { hasText: value }).click();
+}
+
```
Comment (aarondr77) [inline tests/streamlit_ui_tests/utils.ts:59]: <3 love

File: tests/streamlit_ui_tests/features/column_control_panel.spec.ts [hunk 2]
```diff
@@ -66,7 +62,7 @@ test.describe('Column Control Panel', () => {
         
         await mito.getByText('Add Filter').click();
         await mito.getByText('Add a Group').click();
-        await mito.locator('.spacing-row', { hasText: 'Where' }).locator('input').fill('7');
+        await fillInput(mito, 'Where', '7');
```
Comment (aarondr77) [inline tests/streamlit_ui_tests/features/column_control_panel.spec.ts:65]: Want to update the code config, import tests to use this too?

File: tests/streamlit_ui_tests/utils.ts [hunk 3]
```diff
@@ -48,6 +48,15 @@ export const checkColumnCellsHaveExpectedValues = async (mito: FrameLocator, col
...
 }
 
+export const fillInput = async (mito: FrameLocator, title: string, value: string): Promise<void> => {
+    await mito.locator('.spacing-row', { hasText: title }).locator('input').fill(value);
+}
+
+export const updateSelectedValue = async (mito: FrameLocator, title: string, value: string): Promise<void> => {
+    await mito.locator('.spacing-row', { hasText: title }).locator('.select-text').click();
+    await mito.locator('.mito-dropdown-item', { hasText: value }).click();
+}
```
Comment (aarondr77) [inline tests/streamlit_ui_tests/utils.ts:58]:
```suggestion
export const fillInput = async (mito: FrameLocator, inputLabel: string, value: string): Promise<void> => {
    await mito.locator('.spacing-row', { hasText: inputLabel }).locator('input').fill(value);
}

export const updateSelectedValue = async (mito: FrameLocator, selectLabel: string, value: string): Promise<void> => {
    await mito.locator('.spacing-row', { hasText: selectLabel }).locator('.select-text').click();
    await mito.locator('.mito-dropdown-item', { hasText: value }).click();
}
```

Comment (aarondr77) [reply tests/streamlit_ui_tests/utils.ts:58]: Maybe this is clearer to differentiate between title tags of elements (which these inputs and selects often have) vs the label in the row

**Review by aarondr77 — COMMENTED** [review]
Love it. We can move more tests over to this infrastructure and then LGTM once tests pass

File: tests/streamlit_ui_tests/features/column_control_panel.spec.ts (hunk 2 above)
Comment (marthacryan) [reply tests/streamlit_ui_tests/features/column_control_panel.spec.ts:65]: I will when it's merged :)

**Pull request merged**